core/osrd-reporting/src/main/java/fr/sncf/osrd/reporting/Reflection.java

```diff
@@ -13,6 +13,7 @@
 import java.util.Arrays;
 import java.util.Iterator;
 import java.util.List;
+import java.util.Map;
 
 public class Reflection {
```

Comment:
The whole class can be dropped: it's only used to make a polymorphic warning adapter, and the warnings aren't polymorphic anymore

Reply:
Done